libs/@blockprotocol/graph/src/stdlib/subgraph/builder.ts

```diff
@@ -65,11 +72,20 @@ export const buildSubgraph = (
     revisionId: rootRecordId.editionId,
   }));
 
-  const subgraph: Subgraph = {
+  // @ts-expect-error -- @todo, how do we convince TS that we're only setting this when the generic is satisfied
+  const subgraph: Subgraph<Temporal, EntityRootType<Temporal>> = {
```

Comment:
Didn't want to sink much time into this but completely ignoring `tsc` here is also not great.. I haven't found a good way to inform TSC that the generic has been narrowed at certain points within function bodies. This is a recurring inconvenience and in the following PR there will be many places where it's something like `if (subgraph.temporalAxes !== undefined)`... and it would be really great if there was a way to then tell TS "hey `Temporal` is actually `true` here"